src/App.css

```diff
@@ -1,11 +1,39 @@
+.calculator {
+  display: flex;
+  flex-direction: column;
+  width: 700px;
+  border: 1px solid black;
+}
+
```

Comment:
it's not in requirements, but you can make the calculator display to occupy the middle 
`margin:auto`

Reply:
Thanks!